packages/collector/Collector.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * @package Collector
+ * @version 0.0.0
+ */
+/*
+Plugin Name: Collector
+Plugin URI: https://github.com/seanmorris/collector
+Description: Packages your WordPress install and sends it to Playground.
+Author: Sean Morris
+Version: 0.0.0
+Author URI: https://github.com/seanmorris/
```

Comment:
As this plugin is the exploratory ground for a canonical plugin and then merging in core, let's go with Author: WordPress Contributors to communicate it's a collective effort and invite new folks to participate. I'll be adjusting the same thing in the Playground Block plugin also hosted in this repo.